Essentials/src/com/earth2me/essentials/EssentialsPlayerListener.java

```diff
@@ -68,9 +69,22 @@ private static boolean isCommandSendEvent() {
         }
     }
 
+    private static boolean isArrowPickupEvent() {
+        try {
+            Class.forName("org.bukkit.event.player.PlayerPickupArrowEvent");
```

Comment:
When was this event added?

Reply:
In the middle of the 1.9 or 1.10 lifecycle

Tridents were added in 1.13 so we don't need a backport for 1.8

Reply:
@md678685 Looking through various API revisions, it looks like this dates as far back as at least 1.9.4 ([source](https://helpch.at/docs/1.9.4/org/bukkit/event/player/PlayerPickupArrowEvent.html)).

Reply:
Is there an equivalent for 1.8.8 to limit picking up arrows? If not, this isn't a blocker.

Reply:
I added 1.8.8 support, tested and works.